pkg/files/file_helpers.go

```diff
@@ -26,10 +26,11 @@ func GetFileMeta(filePath string) (*v1.FileMeta, error) {
 		return nil, err
 	}
 
-	hash, err := GenerateFileHash(filePath)
+	content, err := ReadFile(filePath)
 	if err != nil {
 		return nil, err
 	}
+	hash := GenerateHash(content)
```

Comment:
I think we are now generating a uuid from the file hash instead of from the content of the file, not sure if that correct